### DISCUSSION TITLE
Copyright text when launching terminal.

### DISCUSSION THREAD

**Discussion by narcomerk19** [chrisant996/clink#376] [Q&A] [answered]
Hello,

Is there any other way to hide those copyright text when starting the terminal. I respect your creation but hopefully there's an option in hiding it. I can't find it in clink script. Hoping for response. Thanks!

**Comment by pukkandan** [comment, marked as answer]
See https://chrisant996.github.io/clink/clink.html#:~:text=clink.logo

```
❯ clink set clink.logo
        Name: clink.logo
 Description: Controls what startup logo to show
     Options: none,full,short
       Value: none

The default is 'full' which shows the full copyright logo when Clink is
injected.  A value of 'short' shows an abbreviated startup logo with version
information.  A value of 'none' omits the startup logo entirely.
```

**Reply by narcomerk19** [reply]
Thanks for fast response. I already done it and it works.

**Comment by chrisant996** [comment]
P.S.  the copyright message is there for legal reasons, not as vanity.  The version and web address are there for troubleshooting reasons, so that it's clearly possible to tell that Clink is present, what version of Clink is in use, and to find out "what in the world is this Clink thing" since sometimes people share a computer and some of the people sharing it might not have heard or been told that Clink had gotten installed.

That's why it's shown as `full` by default.  Personally, I set it to `short` so that I can always see what version of Clink is in use.
